.travis.yml

```diff
@@ -88,6 +88,7 @@ jobs:
     # Ubuntu Linux with glibc using g++-5
     - stage: Linter + Doxygen + non-debug Ubuntu/gcc-5 test
       os: linux
+      dist: xenial
```

Comment:
Do we still need `sources: ubuntu-toolchain-r-test` with Xenial?

Reply:
I can experiment with removing that, and other sources, but only once CI passes. I expect there will be a few things to sort out to get it to pass.